simulation.cpp

```diff
@@ -536,14 +542,14 @@ void test_simulation() noexcept//!OCLINT test may be many
     }
 #endif
 
-//#define FIX_ISSUE_24
+#define FIX_ISSUE_24
 #ifdef FIX_ISSUE_24
     {
         simulation s;
         create_inputs(s);
-        assign_inputs(s);
-        assert(s.get_env_inputs() == s.get_inds_inputs()/*if you have a function like this with a different name put it here*/);
-
+        assign_inputs(s); //would update_inputs not be a better name? rn this overloads another function in a confusing way
```

Comment:
what is the other function? The one that takes a `population` as an argument? I think they more or less do the same thing no?

Reply:
But ofc you cna rename it if you want.

Reply:
ah ok I see, you maybe mean `assign_new_inputs`, mmmm, your call... eheh